src/module-2/components/stylesheets/layout.css

```
@@ -0,0 +1,160 @@
+.filters {
+  grid-area: filt;
+  display: flex;
+  flex-direction: column;
+  gap: 15rem;
+}
+
+@media (max-width: 560px) {
+  .filters {
+    display: none;
+  }
+}
+
+.caption {
+  font-size: 18rem;
+  font-weight: 400;
+  display: flex;
+  align-items: baseline;
+  justify-content: space-between;
+}
+
+.caption button {
+  height: 40rem;
+  width: 45rem;
+  border-radius: 8rem;
+  border: 0;
+  background: #FFFFFF;
```

minor: consider using shorts hex forms where possible  
```suggestion
  background: #FFF;
```